torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7210,6 +7210,37 @@ def sample_inputs_mse_loss(op_info, device, dtype, requires_grad, **kwargs):
         for shape, kwargs in shapes_and_kwargs
     ]
 
+def sample_inputs_l1_loss(op_info, device, dtype, requires_grad, **kwargs):
+    sample_inputs = sample_inputs_loss(op_info, device, dtype, requires_grad, **kwargs)
+
+    if not dtype.is_complex:
+        return sample_inputs
+
+    make = partial(make_tensor, (), device=device, requires_grad=requires_grad)
+    float_dtype = torch.float64
+
+    sample_inputs.extend(
+        [
+            SampleInput(make(dtype=dtype), args=(make(dtype=float_dtype),)),
+            SampleInput(make(dtype=float_dtype), args=(make(dtype=dtype),)),
+        ]
+    )
+
+    return sample_inputs
+
+def sample_inputs_smooth_l1_loss(op_info, device, dtype, requires_grad, **kwargs):
+    sample_inputs = sample_inputs_loss(op_info, device, dtype, requires_grad, **kwargs)
+
+    make = partial(make_tensor, (S, S), device=device, dtype=dtype, requires_grad=requires_grad)
+    sample_inputs.extend(
+        [
+            SampleInput(make(low=0, high=2), args=(make(low=-2, high=0),), kwargs=dict(beta=5)),
```

Comment:
Let's add a comment that this is the everywhere less than beta case, and the next case is everywhere greater than beta